components/03-organisms/menu/utility-nav/utility-nav.twig

```diff
@@ -0,0 +1,20 @@
+{% set utility_nav__base_class = 'utility-nav' %}
+
+<div {{ bem('utility-bar') }}>
```

Comment:
Why use the bem function here at all if a blockname or modifier will never apply?

Reply:
For consistency, and if I ever do need them, it's just a matter of adding parameters, rather than refactoring `class="utility-bar'` to use the `bem` function.

Reply:
Also, about the `file.includes`... I wonder if you didn't run `npm install` or something? The built demo library seems to be working fine, so I think it must have been a local build steps issue or something

Reply:
no idea what is going on with the `file.includes` thing. I switched branches and switched back to this and it's fine. Scratch this feedback :man_shrugging: